roles/redhat_subscription/tasks/main.yml

```diff
@@ -21,14 +21,22 @@
 # Below we use string formatting to use a variable as the name of the file in
 # the lookup()
 # via http://stackoverflow.com/a/34239020
-#
-  - name: Get origin file
-    command: ostree admin --print-current-dir
-    register: origin_file
 
-  - name: Get refspec from origin file
-    command: grep refspec {{ origin_file.stdout }}.origin
-    register: refspec
+  - name: status of /usr/bin/ostree is known
+    shell: type -Pf /usr/bin/ostree | /bin/true
```

Comment:
This could actually give a false positive if a host has `atomic` installed, which I believe pulls in `ostree` as a dependency.  Even if the dependency is not there, some host *could* have `ostree` installed and could cause the false positive.

You should probably check for `/run/ostree-booted` which *should* only be present on Atomic Host systems.  See https://github.com/projectatomic/atomic-host-tests/blob/master/roles/atomic_host_check/tasks/main.yml#L9